pages/public_cloud/compute/securing_and_structuring_projects/guide.en-gb.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Best Practices for securing & structuring OVHcloud Public Cloud Projects'
+excerpt: 'Learn how to secure your OVHcloud account, manage identities, implement best practices for structuring your Public Cloud projects and use IAM policies to restrict access rights'
+updated: 2024-07-22
+---
+
+## Objective
+
+**This guide explains how to secure your OVHcloud account, manage identities, implement best practices for structuring your Public Cloud projects and use IAM policies to restrict access rights**
+
+## Requirements
+
+- An active OVHcloud account
+- Access to your OVHcloud Control Panel
+
+## Instructions
+
+### Step 1: Securing Your Account
+
+The official "[Securing my OVHcloud account and managing my personal information](/pages/account_and_service_management/account_information/all_about_username)" guide provides general best practices for account management and is a good place to start.
+
+#### Enable Two-Factor Authentication (2FA)
+- **What it does**: Adds an extra layer of security by requiring a second form of verification (e.g., mobile app, security key) in addition to your password.
```

Comment:
For them to be well displayed on prod, bullet points lists and numbered lists must always be preceded by one empty line.